CMakeLists.txt

```diff
@@ -539,9 +539,11 @@ else()
   message(STATUS "Using static fmt from Externals")
   add_subdirectory(Externals/fmt EXCLUDE_FROM_ALL)
 endif()
-add_subdirectory(Externals/glslang)
 add_subdirectory(Externals/imgui)
 
+include_directories(Externals/glslang)
```

Comment:
Does this have to be added as a project-wide include? Seems a little overkill.

Reply:
Its been so long I don't recall why I added this, but it seems to compile/work fine without it so removed, thanks!

Reply:
Now that I see the travis output I remembered. It finds the system glslang on my system and still builds. I'll see if I can fix it...

Reply:
I added the include to `Source/Core/VideoBackends/Vulkan/CMakeLists.txt` and `Externals/glslang/CMakeLists.txt` instead, it should be good now.